### ISSUE TITLE
 Inaccessible interactive elements in various apps

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1645]
You can’t reach everything when using the keyboard. By using ‘tab’ on your keyboard, you should be able to cycle through all interactive
elements on the page. Examples of this include:
• The ‘don’t’ show again cross to close the notification asking for help.
• The buttons for adding items to the Kanban boards are reachable by keyboard, but not usable (note: this does work for another
auditor, the difference might be whether you’re logged in)
• The color theme buttons in the Forms app.
• The grey menu in the top-right corner of questions in the Forms app.
• The Present toggle button in the Code app, Share > Link > Access rights.
• Presenting the Markdown Slides presentation.
• Choosing a different color in the Whiteboard app
